modules/juce_gui_basics/desktop/juce_Displays.h

```diff
@@ -73,6 +73,14 @@ class JUCE_API  Displays
             pixels per inch, divide this by the Display::scale value.
         */
         double dpi;
+
+#if JUCE_MAC

```

Comment:
I think there's no harm on having the member exist in all platforms.
This way one could log it without needing to `ifdef`